### ISSUE TITLE
False positive for macros

### ISSUE THREAD

**Issue by dev-ardi** [bnjbvr/cargo-machete#130]
If a macro from another crate expands to use a dependency this falsly detects it.

This tool should expand all macros before running.

**Comment by bnjbvr** [comment]
Yes, this is a known shortcoming of the tool. Are you willing to work on a patch?

**Comment by dev-ardi** [comment]
No, as I've found that the `unused_crate_dependencies` lint works better for me.
